src/datajudge/constraints/numeric.py

```diff
@@ -158,3 +158,55 @@ def test(self, engine: sa.engine.Engine) -> TestResult:
         )
         result = deviation <= self.max_absolute_deviation
         return TestResult(result, assertion_text)
+
+
+class NumericPercentile(Constraint):
+    def __init__(
+        self,
+        ref: DataReference,
+        k: float,
+        max_absolute_deviation: Optional[float],
+        max_relative_deviation: Optional[float],
+        name: str = None,
+        *,
+        ref2: DataReference = None,
+        expected_percentile: float = None,
+    ):
+        super().__init__(
+            ref,
+            ref2=ref2,
+            ref_value=expected_percentile,
+            name=name,
+        )
+        if k < 0 or k > 100:
```

Comment:
```suggestion
        if 0 > k > 100:
```

Reply:
I guess I accepted to quickly :D

Reply:
Or I suggested too quickly. I'm missing a `not` 

```if not (0 > k > 100):```